app/src/main/java/nl/tudelft/cs4160/trustchain_android/main/TrustChainActivity.java

```diff
@@ -128,6 +128,13 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 Intent chainExplorerActivity = new Intent(this, ChainExplorerActivity.class);
                 startActivity(chainExplorerActivity);
                 return true;
+            case R.id.close:
+                if (Build.VERSION_CODES.KITKAT <= Build.VERSION.SDK_INT) {
+                    ((ActivityManager) getApplicationContext().getSystemService(ACTIVITY_SERVICE))
+                            .clearApplicationUserData();
+                } else {
+                    Toast.makeText(getApplicationContext(), "Requires at least API 19 (KitKat)", Toast.LENGTH_LONG).show();
```

Comment:
So you can only empty the case when the version is at least Kitkat? Otherwise it will be emptied by Android eventually?